plugins/digitalocean/resoto_plugin_digitalocean/__init__.py

```diff
@@ -53,3 +74,11 @@ def add_args(arg_parser: ArgumentParser) -> None:
             default=[],
             nargs="+",
         )
+        arg_parser.add_argument(
+            "--digitalocean-spaces-access-keys",
+            help="DigitalOcean Spaces access keys for the teams to be collected",
+            dest="digitalocean_spaces_access_keys",
```

Comment:
One should define access and secret keys with this argument? 
How do you know which key to use for which space?

Maybe there is a way to define a safe delimiter so we could use something like `--space-access <access_key><delimiter><secret_key>` (e.g. `--space-access foo:bla`)?